src/pages/relicans.js

```diff
@@ -4,344 +4,115 @@ import SEO from '../components/Seo';
-    },
-    {
-      name: "Mia Moore",
-      pronouns: "they/them",
-      avatar: miaAvatar,
-      bio: <p>Mia Moore (they/them) is a Developer Programs Manager at New Relic. Before joining New Relic, Mia was a Developer Advocate at IBM and has a background in content creation, marketing, and community management. They are enthusiastic about good storytelling in digital content. In their free time, Mia enjoys knitting, birding, and gaming.</p>,
-      socials: [
-        {
-          name: "twitter",
-          url: "https://twitter.com/xomiamoore"
-        },
-        {
-          name: "twitch",
-          url: "https://www.twitch.tv/xomiamoore",
-        },
-        {
-          name: "instagram",
-          url: "https://www.instagram.com/xomiamoore"
-        },
-        {
-          name: "linkedin",
-          url: "https://www.linkedin.com/in/miamoore"
-        }
-      ]
-    },
-    {
-      name: "Pachi Carlson",
-      pronouns: "she/her",
-      avatar: pachiAvatar,
-      bio: <>
-            <p>Pachi is a self-directed learner, who started her career as a Front-End developer and is passionate about HTML and CCS. She is originally from Brazil where she was born and raised, but now she lives in the USA with her husband, cats and bearded dragon.</p>
-            <p>Very active online, Pachi felt in love with the community and even being a junior herself, she began teaching what she knew and sharing her learning Journey writing and live coding on Twitch, and that was how she got interested in Developer Relations. Now a Developer Relations Engineer at New Relic.</p>
-          </>,
-      socials: [
-        {
-          name: "twitter",
-          url: "https://twitter.com/pachicodes"
-        },
-        {
-          name: "twitch",
-          url: "https://twitch.tv/pachicodes",
-        },
-        {
-          name: "instagram",
-          url: "https://instagram.com/pachicodes"
-        },
-        {
-          name: "linkedin",
-          url: "https://linkedin.com/in/pachicodes"
-        }
-      ]
-    },
-    {
-      name: "Rachael Wright-Munn",
-      pronouns: "she/her",
-      avatar: rachaelAvatar,
-      bio: <p>Rachael's been building software and web apps since 2012. Now she builds her open-source vue/rails app live on Twitch. Enough people thought that was cool, so she's been invited here.</p>,
-      socials: [
-        {
-          name: "twitter",
-          url: "https://twitter.com/ChaelCodes"
-        },
-        {
-          name: "twitch",
-          url: "https://twitch.tv/ChaelCodes",
-        },
-        {
-          name: "linkedin",
-          url: "https://www.linkedin.com/in/rachael-wright-chaelcodes/"
-        }
-      ]
-    }
-  ]
-
-  const teamMemberPanels = teamMembers.map(teamMember => {
-    const socialItems = teamMember.socials.map(socialNetwork => {
+  const teamMemberPanels = teamMembers.map((teamMember, i) => {
+    const socialItems = teamMember.socials.map((socialNetwork, i) => {
```

Comment:
It looks like you're using `i` twice, both for the outer loop and the inner one. Does that cause any problems? Would it make sense to update one of them to `j`?

Reply:
that does make more sense :) I will fix that, I was just getting some linting errors for not having a key so my immediate reaction is to add an i with a key